x-pack/plugin/core/src/main/resources/fleet-actions.json

```diff
@@ -22,6 +22,12 @@
         "expiration": {
           "type": "date"
         },
+        "start_time": {
+          "type": "date"
+        },
+        "minimum_execution_duration": {
+          "type": "keyword"
```

Comment:
@michel-laterman What is the type of this field? duration looks more like a integer to me?

Reply:
I was intending it to be a duration string i.e. `10m`
i was looking at a more recent draft of the json-schema we use on the fleet-server: https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1

Let me know if I should change it to an integer

Reply:
So this would be a `Time.Duration`, I would just use an integer value here so we are aligned with the value of timeout which I believe is in seconds?